src/openrct2/actions/GameAction.h

```diff
@@ -156,6 +160,12 @@ class GameActionParameterVisitor
     {
     }
 
+    void Visit(CoordsXY& param)
+    {
+        Visit("x", param.x);
+        Visit("y", param.y);
+    }
```

Comment:
Does `Visit` modify the `CoordsXY`?

Reply:
Yes, this function is used to get and set the parameters.